docs/lua-api/classes/uenum.md

```diff
@@ -12,6 +12,25 @@
 
 ### ForEachName(LuaFunction Callback)
 
-- Iterates every `FName`/`Value` combination that belongs to this enum.
+- Iterates every `FName`, `Value` combination that belongs to this enum.
```

Comment:
```suggestion
- Iterates every `FName`/`Value` combination that belongs to this enum.
```
The `/` was appropriate here and is more clean than a `,` for this usage of the pair.